.github/workflows/cypress.yml

```diff
@@ -3,23 +3,26 @@ name: E2E test
 on:
   push:
     branches:
-      - main
+    - main
   pull_request:
     branches:
-      - develop
+    - develop
 jobs:
   cypress:
-    name: ${{ matrix.core.name }}
+    name: ${{ matrix.job_name }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        core:
-          - {name: 'WP latest', version: 'latest'}
-          - {name: 'WP minimum', version: 'WordPress/WordPress#5.7'}
-          - {name: 'WP trunk', version: 'WordPress/WordPress#master'}
```

Comment:
Objects are invalid matrix elements.

Reply:
If this is removed, how will GH Actions know which versions to test against? This appears to be working fine: https://github.com/psorensen/curated-query-loop/actions/runs/7579645133

Reply:
Yes it appears to work but not supported.
A proper matrix is added in the next lines.

Reply:
Oops! I missed this. Thanks!